src/Ocelot/Headers/AddHeadersToRequest.cs

```diff
@@ -68,11 +68,11 @@ public void SetHeadersOnDownstreamRequest(IEnumerable<AddHeader> headers, HttpCo
                         continue;
                     }
 
-                    requestHeader.Add(header.Key, new StringValues(value.Data));
+                    requestHeader.Append(header.Key, new StringValues(value.Data));
```

Comment:
Why did you include this change?

Reply:
[it's now a warning in .net8, asp0019](https://learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0019?view=aspnetcore-8.0)